### PR TITLE
HAMLET: final version.

### PR DESCRIPTION
**Pull request by djoerch**
NOTE: There are still formatting problems for many equations.

If you have any suggestion for fixing those, please let me know! :)

### CODE REVIEW

File: machine-learning/_posts/2018-08-08-HAMLET.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+   $$R_m^j(\mathbf{r}) = r^jY^j_m(\mathbf{n})$$ (where $$r = |\mathbf{r}|$$
+   and $$\mathbf{n} = \frac{\mathbf{r}}{|\mathbf{r}|}$$), they further
+   define a *spherical differential operator*
+
+   $$\mathbf{\partial}^j := (\partial^j_{-j}, ..., \partial^j_{j})$$,
+
+   where $$\partial^j_m := R^j_m(\nabla)$$ on the basis of spherical
+   products.
+
+   The authors emphasise that this *"operator can be used to linearly
```
Comment (CharlesAuthier) [inline machine-learning/_posts/2018-08-08-HAMLET.md]: I am maybe wrong but it emphasizes?

File: machine-learning/_posts/2018-08-08-HAMLET.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+   map" spherical tensor fields "onto other" spherical tensor fields
+   "by intertwining spatial neighbourhood information"*.
```
Comment (CharlesAuthier) [inline machine-learning/_posts/2018-08-08-HAMLET.md]: neighborhood, no "u"

File: machine-learning/_posts/2018-08-08-HAMLET.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+ Euler integration is used to obtain streamlines from HAMLET output.
+ Thresholding of the output is performed by "optimising the segmentation
+ accuracy".
+
+ The comparison of tractography output for HAMLET and the ground truth
+ a random test subject from the PRISMA data set is used. The authors
+ find their results overall visually appealing. However, they acknowledge
+ artifacts (yellow circles in the figure below) in the results of their
+ own method. On source of error is high variability in the ground truth
```
Comment (CharlesAuthier) [inline machine-learning/_posts/2018-08-08-HAMLET.md]: I don't understand the phrasing of "on source of error is ...."

Comment (djoerch) [reply machine-learning/_posts/2018-08-08-HAMLET.md]: Typo: "One source of ..."

Anyways, I rephrased that sentence a bit.

File: machine-learning/_posts/2018-08-08-HAMLET.md (hunk 2 above)
Comment (djoerch) [reply machine-learning/_posts/2018-08-08-HAMLET.md]: I try to use British English. Is that OK?

File: machine-learning/_posts/2018-08-08-HAMLET.md (hunk 1 above)
Comment (djoerch) [reply machine-learning/_posts/2018-08-08-HAMLET.md]: If you refer to the spelling: it should be British.

File: machine-learning/_posts/2018-08-08-HAMLET.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+
+ The below figure shows test results of three subjects on the TRIO
+ data set for HAMLET after training on PRISMA for the two acquisition
+ time points.
+
+ <p style="text-align:center"><img src="/machine-learning/images/HAMLET/retest_trio.png" width="700"></p>
+
+ In order to quantify the retest reproducibility, they assess the
+ "apparent tract volume" computed on the output of HAMLET which they
+ define as "the squared power" of the output map $$\mathbf{y}$$ given by
```
Comment (jodoin) [inline machine-learning/_posts/2018-08-08-HAMLET.md]: $$\mathbf{y}$$  does not render

File: machine-learning/_posts/2018-08-08-HAMLET.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+ $$\int |\mathbf{y}(\mathbf{r})|^2 \mathrm{d}\mathbf{r}$$.
```
Comment (jodoin) [inline machine-learning/_posts/2018-08-08-HAMLET.md]: this line does not render well

**Pull request merged**